src/main/java/wolox/training/exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND, reason = "User not found")
+public class UserNotFoundException extends Exception {
+
+    public UserNotFoundException() {
+        super();
```

Comment:
Same as previous PRs

Reply:
Uppps, a rebase don't work that I expect :sweat_smile: